src/Structures/TransformedType.php

```diff
@@ -87,4 +92,16 @@ public function replaceSymbol(string $class, string $replacement): void
             $this->transformed
         );
     }
+
+    public function toString() {
+        switch ($this->format) {
+            case 'enum': {
+                return "enum {$this->name} { {$this->transformed} };";
+            }
+
+            default: {
+                return "type {$this->name} = {$this->transformed};";
+            }
+        }
```

Comment:
Since this package is >= PHP 8, we can use `match` instead of `switch` here:

```suggestion
        return match ($this->format) {
            'enum' => "enum {$this->name} { {$this->transformed} };",
            default => "type {$this->name} = {$this->transformed};",
        };
```